dll/win32/browseui/internettoolbar.cpp

```diff
@@ -250,11 +250,11 @@ HRESULT CDockSite::GetRBBandInfo(REBARBANDINFOW &bandInfo)
         bandInfo.fStyle |= RBBS_BREAK;
     if (fDeskBandInfo.dwModeFlags & DBIMF_TOPALIGN)
         bandInfo.fStyle |= RBBS_TOPALIGN;
-    if (fFlags & ITF_NOGRIPPER || fToolbar->fLocked)
+    if (fFlags & ITF_NOGRIPPER || (fToolbar->pSettings && fToolbar->pSettings->fLocked))
         bandInfo.fStyle |= RBBS_NOGRIPPER;
     if (fFlags & ITF_NOTITLE)
         bandInfo.fStyle |= RBBS_HIDETITLE;
-    if (fFlags & ITF_GRIPPERALWAYS && !fToolbar->fLocked)
+    if (fFlags & ITF_GRIPPERALWAYS && (fToolbar->pSettings && !fToolbar->pSettings->fLocked))
```

Comment:
I wonder whether this code can be called with the `fToolbar->pSettings` being NULL while `fToolBar` being "valid" (not NULL) ....
Otherwise adding an `ASSERT(fToolbar && fToolbar->pSettings);` to inform that this cannot happen _normally_ may be considered :thinking:

Reply:
This check is no longer needed now that the settings for the InternetToolbar gets set on `SetSite()`